index.bs

```diff
@@ -6238,8 +6234,25 @@ The [=DefaultLocale=] algorithm is implementation defined. A WebDriver-BiDi
     1. If [=locale overrides map=] [=map/contains=] |user context|, return
        [=locale overrides map=][|user context|].
 
-1. Return the result of implementation-defined steps in accordance with the
-   requirements of the [=DefaultLocale=] specification.
+1. Return null
+
+</div>
+
+<div algorithm="updated DefaultLocale steps">
```

Comment:
EcmaScript's [6.2.3 DefaultLocale ( )](https://tc39.es/ecma402/#sec-defaultlocale):
> The returned value is is a potential fingerprinting vector. In browser environments, it should match [navigator.language](https://html.spec.whatwg.org/#language-preferences) to avoid providing any additional distinguishing information.

I believe this algorithm "updated DefaultLocale steps" can be removed after the HTML's [`language`](https://html.spec.whatwg.org/#dom-navigator-language) will respect the "WebDriver BiDi emulated language" (after https://github.com/whatwg/html/pull/11793 is merged). Could you please add a note with TODO to remove it?